cluster-stamp.bicep

```
@@ -1411,6 +1413,44 @@ resource paManagedIdentitiesEnabled 'Microsoft.Authorization/policyAssignments@2
   }
 }
 
+// Deploying and applying the custon policy 'Kubernetes cluster ingress TLS hosts must have defined domain suffix' as defined in 'custom-policy_K8sCustomIngressTlsHostsHaveDefinedDomainSuffix.bicep'
+// Note: Policy definition must be deployed as module since policy definitions require a targetScope of 'subscription'.
+
+module modK8sIngressTlsHostsHaveDefinedDomainSuffix 'nested_K8sCustomIngressTlsHostsHaveDefinedDomainSuffix.bicep' = {
+  name: 'modK8sIngressTlsHostsHaveDefinedDomainSuffix'
+  scope: subscription()
+  params: {
+    policyName: policyNameIngressTlsHostsSuffix
+  }
+}
+
+resource paK8sIngressTlsHostsHaveSpecificDomainSuffix 'Microsoft.Authorization/policyAssignments@2021-06-01' = {
+  dependsOn: [
+    modK8sIngressTlsHostsHaveDefinedDomainSuffix
+  ]
+  name: guid(policyNameIngressTlsHostsSuffix, resourceGroup().id, clusterName)
```

Depends on will already be added based on the existing references to the `outputs`.

```suggestion
  name: guid(modK8sIngressTlsHostsHaveDefinedDomainSuffix.outputs.policyId, resourceGroup().id, clusterName)
```